### DISCUSSION TITLE
Contests or challenges?

### DISCUSSION THREAD

**Discussion by Cjjuntti** [keiwando/toonsquid#645] [General]
Toonsquid is probably my favorite app for animation, not just on iPad, but in general, and after using a computer for animation, that’s saying something. Though, I think that the official toonsquid channel should host challenges/contests kind of like FlipaClip and Callipeg do. Doing so I think would help in showing what this small studio app is capable, and I bet more people would buy it, the app would make more money, and we could continue to get more challenges or contests this way. With how powerful the app is, it makes me wonder why we haven’t gotten any challenges or contests for it yet. I bet it would the app grow even further.

**Comment by Lioncub901** [comment]
Hi I run the ToonSquid Community accounts. I was thinking of doing challenges but don’t have enough people following to do it if you want to do it then probably follow the ToonSquid Community Instagram or Twitter.

**Reply by Lioncub901** [reply]
I feel 100 followers would probably be the number to start challenges.

**Reply by budaloops** [reply]
Followed on X

**Reply by Gottafindaname** [reply]
There are already too many challenges. We dont need another honestly….
